specifications/variants/transformations/specification_transformation.py

```
@@ -0,0 +1,56 @@
+
+
+class SpecificationTransformation(ABC):
+    """Represent a transformation of a specification.
+
+    Attributes:
+        parser (Parser[CBMCAst]): A parser for CBMC ASTs.
+    """
+
+    _parser: Parser[CBMCAst]
+
+    def __init__(self):
```

Ruff (0.15.1)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)